### ISSUE TITLE
Support Output Padding for ConvTranspose2d in Model Speedup

### ISSUE THREAD

**Issue by haoshuai-orka** [microsoft/nni#4798]
**Describe the issue**:
Hi, I am trying pruning with nni. I found that currently nni doesn't take the output_padding argument of ConvTranspose2d into consideration, which might lead to a shape mismatch.

**Environment**:
- NNI version: 2.6.1
- Training service (local|remote|pai|aml|etc): remote
- Server OS (for remote mode only): Linux
- Python version: 3.8
- PyTorch/TensorFlow version: 1.11.0
- Is conda/virtualenv/venv used?: Yes
- Is running in Docker?: No

**Configuration**:
![image](https://user-images.githubusercontent.com/85916223/164964680-69224cda-205d-4f39-b29e-50c6cf23dcb4.png)

**Log message**:
![image](https://user-images.githubusercontent.com/85916223/164964566-c789be29-3934-4d99-8616-7457fb62f101.png)

BTW, I fixed the problem by modifying the replace_convtranspose2d function in nni/compression/pytorch/speedup/compress_modules.py

![image](https://user-images.githubusercontent.com/85916223/164964730-8e2e4cc8-c7c7-4b8a-b16c-6411f935a4cb.png)

Is there anything else that I should modify? Thank you!

**Comment by zheng-ningxin** [comment]
Nice catch! Thanks for your valuable feedback! Please submit a pr to fix this problem if you have time.

**Comment by haoshuai-orka** [comment]
I submitted one. Thanks :)

**Comment by Louis-J** [comment]
Thanks for contribution ; )
